docs/gettinginvolved/example-acceptance.yml

```diff
@@ -45,3 +50,7 @@ storage-drivers:
     username: "your.rackspace.username"
     api_key: "33333333333333333333333333333333"
     auth_url: "https://identity.api.rackspacecloud.com/v2.0"
+
+  gce:
+    project: "your-project-name"
+    zone: "us-central1-f"
```

Comment:
It is in the prod case (where this is only consumed by dataset agents), but this is the `acceptance.yml`, so it is also consumed the trial process on your laptop which needs project/zone because it also attempts to interact with GCE block devices.